yogstation/code/game/gamemodes/objective_items.dm

```diff
@@ -0,0 +1,5 @@
+/datum/objective_item/steal/holotool
+	name = "the holotool."
+	targetitem = /obj/item/holotool
+	difficulty = 5
+	excludefromjob = list("Research Director")
```

Comment:
the RD already has 2 steal-objectives, not including the research, why this as well?

Reply:
Rd really only has one steal objective his armor I wouldnt really count the card a steal objective for the rd cause theres tons of other places you can get it without going threw the rd